packages/wp-story-editor/src/api/statusCheck.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { base64Encode } from '@web-stories-wp/story-editor';
```

Comment:
This base64Encode function should be remove to this package. It is only used in the API provider is linked to the WordPress implementation.

Reply:
Done in https://github.com/google/web-stories-wp/pull/8966/commits/57e6c7e5626e2065dee39537c42f60881807558f